src/test/java/org/dependencytrack/resources/v1/VulnerabilityResourceTest.java

```diff
@@ -699,7 +665,7 @@ public void createVulnerabilityWithBadOwaspVectorTest() {
     public void createVulnerabilityCwePreV450CompatTest() {
```

Comment:
I think this entire test case can be deleted now, since it's sole purpose was to ensure behavior of the `cwe` field which we now removed.